pandas/core/resample.py

```diff
@@ -1053,9 +1059,15 @@ def _downsample(self, how, **kwargs):
         how = self._is_cython_func(how) or how
         ax = self.ax
 
+        # Empty groupby objs are accepted only callable funcs
```

Comment:
what triggers this?

Reply:
hmm this is odd. can you remove this fix, and instead: create a new issue which demonstrates this (is this *only* with this method?) and xfail the test. I think something bigger is going on here. We can merge this then come back with a battery of tests for this.

Reply:
@jreback 

to more precisely, this [block](https://github.com/pandas-dev/pandas/blob/ac64c8e6049b438e8f59c129115c450f226c5ffa/pandas/core/groupby/groupby.py#L668-L677) doesn't pass the check and raises exception.
It became to fail after adding `kwargs` to [_groupby_and_aggregate](https://github.com/pandas-dev/pandas/pull/22304/files#diff-28a4430519696cd7934b5e6a5c54feb6R1070)

Reply:
hmm, this is old logic. can you try calling 

``def _try_aggregate_string_function(self, arg, *args, **kwargs)`` (its defined in the base class) and see if this works?

Reply:
No, it doesn't

```
.../pandas/pandas/core/groupby/generic.py in _cython_agg_general(self, how, alt, numeric_only, min_count)
     74                             min_count=-1):
     75         new_items, new_blocks = self._cython_agg_blocks(
---> 76             how, alt=alt, numeric_only=numeric_only, min_count=min_count)
     77         return self._wrap_agged_blocks(new_items, new_blocks)
     78

.../pandas/pandas/core/groupby/generic.py in _cython_agg_blocks(self, how, alt, numeric_only, min_count)
    147
    148         if len(new_blocks) == 0:
--> 149             raise DataError('No numeric types to aggregate')
    150
    151         # reset the locs in the blocks to correspond to our

DataError: No numeric types to aggregate
```

Reply:
hmm, ok, then let's just xfail this test and pls open another issue. I think this is a bigger issues and needs to be addressed separately.

Reply:
@jreback 

Here is the issue - https://github.com/pandas-dev/pandas/issues/22339